.github/workflows/valid-links.yml

```diff
@@ -4,18 +4,33 @@ on:
   push:
   pull_request:
   schedule:
-    - cron: '0 3 * * *'
+    - cron: "0 3 * * *"
 
 jobs:
   verify-links:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@master
+      - run: |
+          git fetch --prune origin master
+          git checkout $GITHUB_SHA
```

Comment:
Not sure what this does, are you sure it's required?

Reply:
Yes this is necessary, as I'm diffing against the master branch and by default, all other branches expect for the PR one are deleted. You then get this error:

> fatal: ambiguous argument 'origin/master': unknown revision or path not in the working tree.